community/front-end/ofe/website/ghpcfe/cluster_manager/clusterinfo.py

```diff
@@ -221,9 +222,14 @@ def _prepare_ghpc_partitions(self, part_uses):
         for (count, part) in enumerate(self.cluster.partitions.all()):
             part_id = f"partition_{count}"
             if part.image is not None:
-                instance_image_yaml = f"""instance_image:
-        family: image-{part.image.family}
-        project: {self.cluster.project_id}"""
+                if part.image.source_image_family == "Imported":
+                    instance_image_yaml = f"""instance_image:
+            family: {part.image.family}
```

Comment:
nit: This might be more readable if you pull the conditional outside of the string and just define the string once:

```
                imported_image_fam = part.image.family
                regular_image_fam = f"image-{part.image.family}"
                image_family = imported_image_fam if part.image.source_image_family == "Imported" else regular_image_fam
                instance_image_yaml = f"""instance_image:
        family:{image_family}
        project: {self.cluster.project_id}"""
```

Totally optional, feel free to ignore and close.